### DISCUSSION TITLE
Image Upload not completing

### DISCUSSION THREAD

**Discussion by lucas-inc-query** [requarks/wiki#4835] [Error / Bug Report] [answered]
When trying to upload an asset, it gets to ~20% before failing and displaying the error message "File upload failed.".

**Comment by ddlidded** [comment]
I am having similar issues. This never happened before.

**Comment by lucas-inc-query** [comment]
Here is the error I am seeing:

![MicrosoftTeams-image](https://user-images.githubusercontent.com/89907484/147549482-7de28edc-b46b-4f42-897e-80ab4e4ea376.png)

**Comment by lucas-inc-query** [comment, marked as answer]
Fixed by adding the `write:assets` global permission (https://docs.requarks.io/groups)
